pkgs/applications/science/misc/openmvg/default.nix

```diff
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
     url = "https://www.github.com/openmvg/openmvg.git";
 
     # Tag v1.1
-    rev = "f5ecb48";
+    rev = "refs/tags/${version}";
```

Comment:
Can you build this with `nix-build -A openmvg.src --check`? I think this should be `refs/tags/v${version}`